docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/app-information/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "App & tracker information"
+---
+
+Information about the app and the tracker instance that the events originate from is tracked using:
+
+1. Atomic event properties
```

Comment:
I feel like we are missing an explanation of the out of the box atomic event properties, they appear hidden in fragments like this, sometimes clearly categorised under atomic events like here, sometime under a category called "platform" (for browser related fields) I feel like it would help if we gave an intro and explained what group of fields we are tracking for web and mobile out of the box and which not. If we leave that to the other page I started updating then at least we should link that to the main tracker page and explain our documentation structure at the beginning.